### DISCUSSION TITLE
403 bug fixes

### DISCUSSION THREAD

**Discussion by leela1255** [subdavis/kobo-book-downloader#132] [General]
Hi, I just wanted to ask if there's any progress on getting the various 403 errors fixed? I know there are several bugs raised and several pull requests sitting open, but there doesn't seem to be any movement on any of them for a few weeks. I know it's not necessarily a case of just approving the PRs, as you need to make sure they're fixing the right thing (and/or not breaking anything else), but I'm just asking because everything seems to be stalled right now, and I'm hoping a fix is coming sometime soon. Thanks!

**Comment by subdavis** [comment]
These have been resolved in 0.10.1 hopefully.  Please follow up if not.
